src/Furniture/AbstractFurniture.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Vendor\FurnitureCalculator\Furniture;
+
+abstract class AbstractFurniture
+{
+    /**
+     * @var array
+     */
+    private $sizes;
+
+    /**
+     * @var string
+     */
+    private $color;
+
+    /**
+     * @var string
+     */
+    private $material;
+
+    /**
+     * @param array  $sizes
+     * @param string $color
+     * @param string $material
+     */
+    public function __construct($sizes, $color, $material)
+    {
+        $this->sizes = $sizes;
```

Comment:
you don't have methods getSizes, getColor etc. How can you use this class if it only takes data but returns nothing?